pingcap/tidb
# config: show tidb_force_priority should be same as config

### What problem does this PR solve? 
If the force-priority is set in the config file, it should be the same as the result of `select @@session.tidb_force_priority` when the user has not set it before.

### What is changed and how it works?
Change the default value for tidb_force_priority the same as it is in the config file.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
Change the config file and then start the server.
Check the result of `select @@session.tidb_force_priority`.

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch
